src/gui/uithememanager.cpp

```diff
@@ -42,9 +43,11 @@
 
 namespace
 {
-    const QString ICONS_DIR = QStringLiteral(":icons/");
-    const QString THEME_ICONS_DIR = QStringLiteral(":uitheme/icons/");
-    const QString CONFIG_FILE_NAME = QStringLiteral(":uitheme/config.json");
+    const char DEFAULT_ICONS_DIR[] = ":icons/";
```

Comment:
Also I would leave the slash out and append it only when needed. Otherwise (in general) the code base ends up with uncertainty of the slash existence.

Reply:
slash is used everywhere, I imagined these variables such that *Dir will end with a slash.

Reply:
This problem will go into oblivion as soon as I finish encapsulating path handling.